#!/usr/bin/env python

# set up parameters that we care about
PACKAGE = 'jsk_pcl_ros'

from dynamic_reconfigure.parameter_generator_catkin import *;

from math import pi

gen = ParameterGenerator ()
gen.add("segment_connect_normal_threshold", double_t, 0,
        "threshold of normal to connect clusters", 0.9, 0.0, 1.0)
gen.add("ewma_tau", double_t, 0,
        "tau parameter of EWMA to connect clusters", 0.2, 0.0, 1.0)
gen.add("outlier_threshold", double_t, 0, "outlier threshold", 0.01, 0.0, 0.1)
gen.add("max_iterations", int_t, 0, "maximum iteration", 100, 1, 10000)
gen.add("min_indices", int_t, 0, "maximum iteration", 1000, 1, 10000)

exit (gen.generate (PACKAGE, "jsk_pcl_ros", "LineSegmentCollector"))
